lib/still/compiler/compilation_queue.ex

```diff
@@ -0,0 +1,117 @@
+defmodule Still.Compiler.CompilationQueue do
```

Comment:
So is this one queue per file?

Reply:
Nope, the queue is a single process that receives requests for compilation.

Reply:
Then the subscribers get notified of every change? Or am I understanding it wrong?

Reply:
No, no, like, whenever a file needs to be recompiled, we put it in here. If the `_header.slim` changes and we need to recompile the `index.slime` and the `about.slime`, both of them are put it here.